diff --git a/src/render.c b/src/render.c
index b9fa3903..7454552d 100644
--- a/src/render.c
+++ b/src/render.c
@@ -453,7 +453,7 @@ static void render_con_tabbed(Con *con, Con *child, render_params *p, int i) {
     child->rect.height = p->rect.height;
 
     child->deco_rect.width = floor((float)child->rect.width / p->children);
-    child->deco_rect.x = p->x - con->rect.x + i * child->deco_rect.width;
+    child->deco_rect.x = p->x - con->rect.x + i * child->deco_rect.width - i * 1;
     child->deco_rect.y = p->y - con->rect.y;
 
     /* Since the tab width may be something like 31,6 px per tab, we
